launch.md

```diff
@@ -0,0 +1,150 @@
+# Configuring launch.json for C/C++ debugging
+
+The `launch.json` file is used to configure the debugger in Visual Studio Code. 
+
+Visual Studio Code generates a `launch.json` with almost all of the required information.  To get started
+debugging you need to fill in the `program` field with the path to the executable you plan to debug. This must be specified for 
+both the launch and attach (if you plan to attach to a running instance at any point) configurations.
```

Comment:
This is actually only true for gdb/llb in the attach case. Of course, right now, we have a bug that concord is requiring it for attach.